flake.nix

```diff
@@ -51,7 +51,7 @@
               cargoLock = {
                 lockFile = "${self}/Cargo.lock";
                 outputHashes = {
-                  "dag-0.1.0" = "sha256-ZSPdQ5WLx977p36bIbxhbsp2k57DbFaAemGj5mcKavw=";
+                  "renderdag-0.1.0" = "sha256-ZSPdQ5WLx977p36bIbxhbsp2k57DbFaAemGj5mcKavw=";
                   "quickcheck_arbitrary_derive-0.1.0" = "sha256-kUgApls8Uq07QcDrgOLXmAk08fNaEdQtDrnGhP6ff6w=";
```

Comment:
I'm not an expert of nix. But this might be no longer needed after https://github.com/facebook/sapling/commit/c4390a93cce958fbdd2c14b43d173e1c4c6c5816.

Reply:
Correct. This can be seen from the output of the failed CI run.

Reply:
I fixed that and found the right hash (I think) by running `nix build`.